src/OrchardCore.Modules/OrchardCore.Rules/Drivers/JavascriptConditionDisplayDriver.cs

```diff
@@ -10,11 +9,8 @@ namespace OrchardCore.Rules.Drivers
 {
     public class JavascriptConditionDisplayDriver : DisplayDriver<Condition, JavascriptCondition>
     {
-        private readonly ConditionOperatorOptions _options;
-
-        public JavascriptConditionDisplayDriver(IOptions<ConditionOperatorOptions> options)
+        public JavascriptConditionDisplayDriver()
```

Comment:
Remove?

Reply:
Okay done, as said could have been part of another PR as not related to our current Analyzer rules